packages/web-components/src/toggle-button/toggle-button.styles.ts

```diff
@@ -0,0 +1,105 @@
+import { css } from '@microsoft/fast-element';
+import { styles as ButtonStyles } from '../button/button.styles.js';
+import {
+  colorBrandBackgroundHover,
+  colorBrandBackgroundPressed,
+  colorBrandBackgroundSelected,
+  colorNeutralBackground1Hover,
+  colorNeutralBackground1Pressed,
+  colorNeutralBackground1Selected,
+  colorNeutralForeground1,
+  colorNeutralForeground2BrandHover,
+  colorNeutralForeground2BrandPressed,
+  colorNeutralForeground2BrandSelected,
+  colorNeutralForegroundOnBrand,
+  colorNeutralStroke1,
+  colorNeutralStroke1Hover,
+  colorNeutralStroke1Pressed,
+  colorSubtleBackgroundHover,
+  colorSubtleBackgroundPressed,
+  colorSubtleBackgroundSelected,
+  colorTransparentBackgroundHover,
+  colorTransparentBackgroundPressed,
+  colorTransparentBackgroundSelected,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+// Need to support icon hover styles
+export const styles = css`
+  ${ButtonStyles}
+
+  :host([aria-pressed="true"]) .control {
+    border-color: ${colorNeutralStroke1};
+    background-color: ${colorNeutralBackground1Selected};
+    color: ${colorNeutralForeground1};
+    border-width: ${strokeWidthThin};
+  }
+
+  :host([aria-pressed='true']:hover) .control {
+    border-color: ${colorNeutralStroke1Hover};
+    background-color: ${colorNeutralBackground1Hover};
+  }
+
+  :host([aria-pressed='true']:active) .control {
+    border-color: ${colorNeutralStroke1Pressed};
+    background-color: ${colorNeutralBackground1Pressed};
+  }
+
+  :host([aria-pressed='true'][appearance='primary']) .control {
+    border-color: transparent;
+    background-color: ${colorBrandBackgroundSelected};
+    color: ${colorNeutralForegroundOnBrand};
+  }
+
+  :host([aria-pressed='true'][appearance='primary']:hover) .control {
+    background-color: ${colorBrandBackgroundHover};
+  }
+
+  :host([aria-pressed='true'][appearance='primary']:active) .control {
+    background-color: ${colorBrandBackgroundPressed};
+  }
+
+  :host([aria-pressed='true'][appearance='subtle']) .control {
+    border-color: transparent;
+    background-color: ${colorSubtleBackgroundSelected};
+    color: ${colorNeutralForeground2BrandSelected};
```

Comment:
According to design spec, content color in `subtle pressed` should use colorNeutralForeground2Selected (no brand).
Only the icon should have the brand color.